src/app/clusters/identify-server/tests/TestIdentifyClusterBackwardsCompatibility.cpp

```diff
@@ -27,6 +29,11 @@ using namespace chip::app::Clusters;
 using namespace chip::app::Clusters::Identify;
 using namespace chip::app::Clusters::Identify::Attributes;
 
+// Forward declaration of the callback to emulate ember initialization
+void MatterIdentifyClusterInitCallback(chip::EndpointId endpointId);
+// Stub for DataModelHandler init function
+void InitDataModelHandler() {}
```

Comment:
This one is suspicious ... why is this required? Does this work for cases when we group all tests in one large binary (e.g. emulator tests or zephyr tests)?